packages/server/src/data/addAppVersionToChannelAsync.js

```diff
@@ -0,0 +1,38 @@
+
+
+
+/**
+ * 
+ * @param {*} appVersionId
+ * @param {*} currentUserId
+ * @param {any}
+ * @param {*} minDhisVersion
+ * @param {*} maxDhisVersion
+ * @param {*} knex
+ * @param {*} trx
+ */
+const addAppVersionToChannelAsync = async (appVersionId, currentUserId, channelName, minDhisVersion, maxDhisVersion, knex, transaction) => {
```

Comment:
Then I would suggest that we use parameter destructuring to have type hinting/intellisense working "out of the box" without switching to typescript, what do you think @Mohammer5 ?

The signature would instead be for example:
```
async ({ appVersionId, currentUserId, channelName, minDhisVersion, maxDhisVersion }, knex, transaction)
```

Reply:
This would however prevent the comment below by making a helper function to dump all parameters into a throwed error, if destructured as parameter already

Reply:
I think in that case you can add a [jsdoc block that describes the contents of the options object](http://usejsdoc.org/tags-param.html#parameters-with-properties), should work with IntelliJ/VSC.